src/git-operations.test.ts

```diff
@@ -190,19 +191,7 @@ describe('didPackageChange', () => {
         name: PACKAGES.A.name,
         manifest: { name: PACKAGES.A.name, version: VERSIONS.A },
         dirName: PACKAGES.A.dir,
-        dirPath: '', // just for interface compliance, not relevant
-      }),
-    ).toStrictEqual(true);
-    expect(execaMock).not.toHaveBeenCalled();
-  });
-
-  it('retrieves cached diff on repeat call for tag', async () => {
```

Comment:
I couldn't understand what this test did. It appeared to be a duplicate of the previous test